File: EmployeeHR.Spa/src/app/employee.ts
export class Employee {

    id: number | undefined;
    firstName: string;
    lastName: string;
    socialSecurityNumber: string;
    phoneNumber: string;
    rowVersion: Date;

    constructor() {
        this.id = undefined;
        this.firstName = "";
        this.lastName = "";
        this.socialSecurityNumber = "";
        this.phoneNumber = "";
        this.rowVersion = new Date();
    }

}

File: EmployeeHR.Dto/Employee.cs
using System;

namespace EmployeeHR.Dto
{
    public class Employee : ICloneable, IEquatable<Employee>
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string SocialSecurityNumber { get; set; }
        public string PhoneNumber { get; set; }
        public DateTime RowVersion { get; set; }

        public object Clone()
        {
            Employee other = (Employee)this.MemberwiseClone();
            return other;
        }

        public bool Equals(Employee other)
        {
            bool isEqual = this.Id.Equals(other.Id) &&
                            this.FirstName.Equals(other.FirstName) &&
                            this.LastName.Equals(other.LastName);
            return isEqual;
        }
    }
}